google-cloud-spanner/lib/google/cloud/spanner/transaction.rb

```diff
@@ -302,7 +305,7 @@ def upsert table, *rows
         #
         def insert table, *rows
```

Comment:
can you update the documentation to make it clear that this buffers the mutations in memory.